src/test/ebpf_yaml.cpp

```
@@ -126,18 +126,32 @@ static std::set<string> as_set_empty_default(const YAML::Node& optional_node) {
     return vector_to_set(optional_node.as<vector<string>>());
 }
 
+static std::map<std::string, std::set<std::string>> parse_invariants_to_check(const YAML::Node& case_node) {
```

cppcheck

[style] The function 'parse_invariants_to_check' is never used.

(unusedFunction)